### ISSUE TITLE
Out of focus

### ISSUE THREAD

**Issue by QuantifiableGaming** [JohnHoder/Javascript-Keylogger#4]
Does this work even if the website is running in the background ?

**Comment by JohnHoder** [comment]
What do you mean by "running in the background"?

**Comment by QuantifiableGaming** [comment]
I mean the website is open but the user is actively using another program

**Comment by JohnHoder** [comment]
No, it's javascript, it's range of access is limited to the injected webpage, i.e. one tab :)
